consensus/service/src/byzantine_ledger/worker.rs

```diff
@@ -397,7 +395,7 @@ impl<
             .expect("nominate failed");
 
         if let Some(msg) = msg_opt {
-            (self.send_scp_message)(msg)
+            let _ = self.issue_consensus_message(msg);
```

Comment:
would this be better if the option was managed inside `issue_consensus_message`?

Reply:
I'd prefer to avoid functions that take options in order to "maybe" do work. That feels funky and complex to me, and doesn't really fit with how to do functional programming with Option.

In fact, this if-let-some block could probably just become
```
    msg_opt.map(self.issue_consensus_msg)
```